### PR TITLE
Finished translation of add-react-to-existing-project page

### PR DESCRIPTION
**Pull request by bwv847**
`add-react-to-an-existing-project.md` translation

### CODE REVIEW

File: src/content/learn/add-react-to-an-existing-project.md [hunk 1]
```diff
@@ -1,59 +1,59 @@
 ---
-title: Add React to an Existing Project
+title: Інтеграція React в існуючий проект
 ---
 
 <Intro>
 
-If you want to add some interactivity to your existing project, you don't have to rewrite it in React. Add React to your existing stack, and render interactive React components anywhere.
+Якщо ви хочете додати інтерактивності до вашого наявного проєкту, не потрібно переписувати його на React. Додайте React до вашого наявного стека і ви зможете рендерити інтерактивні React компоненти будь-де.
```
Comment (bkatsevych) [inline src/content/learn/add-react-to-an-existing-project.md:7]:
```suggestion
Якщо ви хочете додати інтерактивності до вашого існуючого проєкту, не потрібно переписувати його на React. Додайте React до вашого наявного технологічного стеку і ви зможете рендерити інтерактивні React компоненти будь-де.
```

Comment (bwv847) [reply src/content/learn/add-react-to-an-existing-project.md]: виправив

File: src/content/learn/add-react-to-an-existing-project.md [hunk 2]
```diff
@@ -146,10 +146,10 @@ root.render(<NavigationBar />);
 
 </Sandpack>
 
-Notice how the original HTML content from `index.html` is preserved, but your own `NavigationBar` React component now appears inside the `<nav id="navigation">` from your HTML. Read the [`createRoot` usage documentation](/reference/react-dom/client/createRoot#rendering-a-page-partially-built-with-react) to learn more about rendering React components inside an existing HTML page.
+Зверніть увагу як оригінальний зміст HTML з `index.html` зберігся, проте ваш власний `NavigationBar` React компоненти тепер зображуються всередині `<nav id="navigation">` з вашого HTML. Читайте [документацію використання `createRoot`](/reference/react-dom/client/createRoot#rendering-a-page-partially-built-with-react), щоб дізнатися більше про рендерінг React компонентів всередині наявної HTML сторінки.
```
Comment (bkatsevych) [inline src/content/learn/add-react-to-an-existing-project.md:149]:
```suggestion
Зверніть увагу як оригінальний зміст HTML з `index.html` зберігся, проте ваш власний `NavigationBar` React компонент тепер зображуються всередині `<nav id="navigation">` з вашого HTML. Читайте [документацію використання `createRoot`](/reference/react-dom/client/createRoot#rendering-a-page-partially-built-with-react), щоб дізнатися більше про рендерінг React компонентів всередині наявної HTML сторінки.
```

File: src/content/learn/add-react-to-an-existing-project.md [hunk 3]
```diff
@@ -146,10 +146,10 @@ root.render(<NavigationBar />);
...
 
-When you adopt React in an existing project, it's common to start with small interactive components (like buttons), and then gradually keep "moving upwards" until eventually your entire page is built with React. If you ever reach that point, we recommend migrating to [a React framework](/learn/start-a-new-react-project) right after to get the most out of React.
+Коли ви впроваджуєте React у наявний проєкт, зазвичай починають з невеликих інтерактивних компонентів (наприклад, кнопок), а потім поступово "підіймаються вгору" до того часу, поки увесь ваш вебсайт не буде побудований з використанням React. Якщо ви досягнете цієї точки, ми рекомендуємо одразу перейти до [фреймворку React](/learn/start-a-new-react-project), щоб найбільш ефективно його використовувати.
```
Comment (bkatsevych) [inline src/content/learn/add-react-to-an-existing-project.md:151]:
```suggestion
Коли ви впроваджуєте React у існуючий проєкт, стандартна практика - починати з невеликих інтерактивних компонентів (наприклад, кнопок), а потім поступово "підійматися вгору" до моменту, поки вся ваша сторінка не буде побудована з використанням React. По досягненню цього, ми рекомендуємо одразу перейти до [фреймворку React](/learn/start-a-new-react-project), щоб найбільш ефективно його використовувати.
```

File: src/content/learn/add-react-to-an-existing-project.md [hunk 4]
```diff
@@ -146,10 +146,10 @@ root.render(<NavigationBar />);
...
 
-## Using React Native in an existing native mobile app {/*using-react-native-in-an-existing-native-mobile-app*/}
+## Використання React Native в існуючому нативному застосунку {/*using-react-native-in-an-existing-native-mobile-app*/}
```
Comment (bkatsevych) [inline src/content/learn/add-react-to-an-existing-project.md:153]:
```suggestion
## Використання React Native в існуючому нативному додатку {/*using-react-native-in-an-existing-native-mobile-app*/}
```

File: src/content/learn/add-react-to-an-existing-project.md [hunk 5]
```diff
@@ -146,10 +146,10 @@ root.render(<NavigationBar />);
...
 
-[React Native](https://reactnative.dev/) can also be integrated into existing native apps incrementally. If you have an existing native app for Android (Java or Kotlin) or iOS (Objective-C or Swift), [follow this guide](https://reactnative.dev/docs/integration-with-existing-apps) to add a React Native screen to it.
+[React Native](https://reactnative.dev/) також може бути інтегрований в існуючи нативні застосунки поступово. Якщо у вас є наявний нативний застосунок для Android (Java або Kotlin) або iOS (Objective-C або Swift), [дотримуйтесь цього посібника](https://reactnative.dev/docs/integration-with-existing-apps), щоб додати React Native екран до нього.
```
Comment (bkatsevych) [inline src/content/learn/add-react-to-an-existing-project.md:155]:
```suggestion
[React Native](https://reactnative.dev/) також може бути інтегрований в існуючі нативні застосунки поступово. Якщо у вас є наявний нативний додаток для Android (Java або Kotlin) або iOS (Objective-C або Swift), [дотримуйтесь цього посібника](https://reactnative.dev/docs/integration-with-existing-apps), щоб додати React Native екран до нього.
```

Comment (bwv847) [reply src/content/learn/add-react-to-an-existing-project.md]: fixed, thanks

**Review by bkatsevych — APPROVED** [review]

**Pull request merged**